Added ⚡️ content to `Getting Started` > `Visual language` (#475)
BitcoinDesign/Guide: guide/getting-started/visual-language.md

Since Bitcoin answers to no central authority, there is no single symbol or logo that can visually represent it, though many try. Here are some of the most recognized attempts to create a universal logo for Bitcoin.

## The Bitcoin symbol

> ### Original symbol

Reviewer: Maybe add the date the core client was launched with this symbol.

Reviewer 2: Do we know this date?

Reviewer 3: Calling this 'The Bitcoin' symbol is not helping IMO. It's a gold coin with 'BC' in the middle.
When people say the Bitcoin symbol today, they mostly refer to the Bitboy update.

Author: @danielnordh I added this heading because I don't like the page jumping from an H1 to an H3 heading. Let me see what I can do to amend this.

Author: Amended in d8b2932
@Bosch-0 I added a tidbit about the launch date
@danielnordh I removed "The Bitcoin symbol" and adjusted the heading levels.
How does that look?

   layout = "float-left"
%}

> </div>

Reviewer 2: Did you intentionally remove the 2-column layout here? If so, seems inconsistent with the paragraph above. If not, just move the closing div a few lines lower, below the text.

Author: Fixed

On November 1, 2010, a user named bitboy proposed a new icon and set of promotional graphics on the Bitcointalk forum ([archive.org](https://web.archive.org/web/20130524191250/https://bitcointalk.org/index.php?topic=1631.0), [bitcointalk.org](https://bitcointalk.org/index.php?topic=1631.0)). This iteration added a flat style, the hex color #F7931A, and a 14-degree rotation of the B. This interpretation of the bitcoin logo is still ubiquitous today.

#### Download Bitcoin Symbol

> [SVG](/assets/images/guide/getting-started/visual-language/bitcoin-symbol.svg){: .button}

Reviewer 2: You could add the download attribute, so clicking the buttons triggers a download directly instead of navigating to the file.

Author: Fixed

</div>

## The Satoshi symbol

> As the price of bitcoin has increased, and the Lightning network has facilitated very small payments (more on that below), it has become common for people to denominate bitcoin with the "satoshi". See the [Units & Symbols](https://bitcoin.design/guide/payments/units-and-symbols/) section for more detailed information about bitcoin units.

Reviewer 4: >  the Lightning network has facilitated very small payments

Not sure what this is meant to refer to? The volume, the  size of the payments (micropayments) or?

Reviewer 2: ```suggestion
As the price of bitcoin has increased, and the Lightning network has facilitated low-value payments (more on that below), it has become common for people to denominate bitcoin with the "satoshi". See the [Units & Symbols](https://bitcoin.design/guide/payments/units-and-symbols/) section for more detailed information about bitcoin units.
```

Author: I intended to mean micropayments, as in small number of satoshis. I think Christoph's change reflects this well.

> There is currently not an agreed upon symbol to represent the satoshi (or if there should even be one), however, many ideas are in circulation.

Reviewer: I think the original is less opinionated personally

> There is currently not an agreed upon symbol to represent the satoshi (or if there should even be one), however, many ideas are in circulation.

Reviewer 2: Also agree that the original is less opinionated. The updated version assumes there should be a symbol, which is not a given.

</div>

## The Satoshi symbol

> As the price of bitcoin has increased, and the [Lightning network](https://bitcoin.design/guide/glossary/#lightning-network) has facilitated low-value payments, it has become common for people to denominate bitcoin with the "satoshi". See the [Units & Symbols](https://bitcoin.design/guide/payments/units-and-symbols/) section for more detailed information about bitcoin units.

Reviewer 2: ```suggestion
As the price of bitcoin has increased, and the [Lightning network]({{ '/guide/glossary/#lightning-network' | relative_url }}) has facilitated low-value payments, it has become common for people to denominate bitcoin with the "satoshi". See the [Units & Symbols]({{ '/guide/payments/units-and-symbols/' | relative_url }}) section for more detailed information about bitcoin units.
```
Using relative URLs.

image = "/assets/images/guide/getting-started/visual-language/satoshi-symbols.png"
retina = "/assets/images/guide/getting-started/visual-language/satoshi-symbols@2x.png"
alt-text = "Sequence of images showing proposals for satoshi symbols"
caption = "References from top left to bottom right, Keith, <a href='http://satbase.org/' target='_blank'>2</a>, <a href='https://www.reddit.com/r/Bitcoin/comments/ku9nl8/design_proposal_for_the_satoshi_symbol/' target='_blank'>3</a>, <a href='https://twitter.com/PLK5280/status/1326347333219606528' target='_blank'>4</a>, <a href='https://www.reddit.com/r/Bitcoin/comments/ku9nl8/design_proposal_for_the_satoshi_symbol/' target='_blank'>5</a>, Jeremiah, Alexa, <a href='https://satsymbol.com/' target='_blank'>8</a>, <a href='https://twitter.com/BitkoYinowsky' target='_blank'>9</a>, <a href='https://twitter.com/sbtlaln/status/1361368643238567951' target='_blank'>10</a>, <a href='https://www.reddit.com/r/Bitcoin/comments/k0e98n/my_proposal_for_the_satish_symbol_still_looks/' target='_blank'>11</a>, and <a href='https://www.reddit.com/r/Bitcoin/comments/ku9nl8/design_proposal_for_the_satoshi_symbol/' target='_blank'>12</a>."
width = 800

> height = 508

Reviewer 2: ```suggestion
height = 504
```

In digital product design, there are times when a minimally expressive approach is what’s needed. For example, when a user makes a transaction, we want them to focus. At other times, illustrations and stronger visual elements are more appropriate, and this is where we can take advantage of the creative freedom Bitcoin affords.

## The Lightning network

> The Lightning network is a layer 2 solution for scaling Bitcoin. It allows people to make almost instant payments with very small fees.

Reviewer 4: We don't need to explain what LN is on every page, this is already covered in the Tech Primer and other pages.
```suggestion
```

Reviewer 3: Added a brief entry in the Glossary which can be linked to when #479 is merged.

Author: Thanks, added glossary link

> Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow icon of a lightning bolt. Sometimes, Lightning software and services may choose to completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.

Reviewer: Wonder if its worth mentioning the distinct visual branding of the various Lightning implementations. Also the purple circle with the bolt in the middle is what I was familiar with from the early Lightning days, it's a primary image on on Wikipedia also: https://commons.wikimedia.org/wiki/File:Bitcoin_over_Lightning_Network.svg

Reviewer 3: Showing a variety of them instead of the various Lightning wallet logos is my recommendation.

Author: c7104e4 I updated this image to include the purple LN icon. I think the purple version actually has been somewhat influential; I feel like purple is the color I see most often associated with LN second to yellow.

@Bosch-0 I don't want to discuss the distinct brandings of different LN implementations -- my hope is that the user never has to worry about things like LND, C-Lightning, or Eclair; and my thinking is that it may be too much detail for a designer who is simply trying to consider the needs of their user.

> Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow icon of a lightning bolt. Sometimes, Lightning software and services may choose to completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.

Reviewer 4: An attempt to simplify the paragraph. Should we mention the use of `emoji` I see a lot of applications using bolt emoji instead of the icons.

```suggestion
Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow lightning bolt icon. Sometimes, Lightning software and services completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.
```

<div class="image-gallery" markdown="1">

{% include image.html
image = "/assets/images/guide/getting-started/visual-language/bitcoin-lightning-branding.jpg"
retina = "/assets/images/guide/getting-started/visual-language/bitcoin-lightning-branding@2x.jpg"
alt-text = "Sequence of images showing examples of logos and brands for lightning software and services"
caption = "References from top top bottom, left to right: <a href='https://www.walletofsatoshi.com/' target='_blank'>1</a>, <a href='https://shop.fulmo.org/raspiblitz/' target='_blank'>2</a>, <a href='https://www.lightningnetworkstores.com/' target='_blank'>3</a>, <a href='https://zeusln.app/' target='_blank'>4</a>, and <a href='https://lightning.store/product/bitcoin-rtl-t-shirt/' target='_blank'>5</a>."
width = 800
height = 600
%}

</div>

## Should Lightning be distinct from Bitcoin?

While Lightning branding can be fun and expressive, it may give new Bitcoin users the impression that Lightning is different from Bitcoin. More accurately, Lightning should be thought of as a different way of using bitcoin, not as a different currency than bitcoin.

#### Consider your audience

> Are you developing Bitcoin products that are intended to be used primarily by developers or people who are very experienced with Bitcoin? If so, lightning branding may be a great choice for your product.

Reviewer 2: I think you brought this up jokingly, but I am not confused about the mixed use of lowercase and uppercase Lightning/lightning. "Consider your audience" is lowercase, which seems to break with the rest of the page.

Author: That's just a typo. I'll correct the casing throughout.

Reviewer 2: Thank you.

Author: Resolved in eb50035

Since the first Lightning implementation in 2018, a distinct visual identity has developed around the Lightning network based on a simple yellow lightning bolt icon. Sometimes, Lightning software and services completely forego common elements of Bitcoin branding (the color orange, the Bitcoin symbol, etc.) and instead use the color yellow, a lightning bolt symbol, and metaphors for electricity.

<div class="image-gallery" markdown="1">

{% include image.html
image = "/assets/images/guide/getting-started/visual-language/bitcoin-lightning-branding.jpg"
retina = "/assets/images/guide/getting-started/visual-language/bitcoin-lightning-branding@2x.jpg"
alt-text = "Sequence of images showing examples of logos and brands for lightning software and services"
caption = "References from top top bottom, left to right: <a href='https://www.walletofsatoshi.com/' target='_blank'>1</a>, <a href='https://shop.fulmo.org/raspiblitz/' target='_blank'>2</a>, <a href='https://www.lightningnetworkstores.com/' target='_blank'>3</a>, <a href='https://zeusln.app/' target='_blank'>4</a>, and <a href='https://lightning.store/product/bitcoin-rtl-t-shirt/' target='_blank'>5</a>."
width = 800

> height = 600

Reviewer 2: ```suggestion
height = 584
```
